Tests/Kernel/TestMemoryDeviceMmap.cpp

```diff
@@ -23,7 +23,12 @@ static ALWAYS_INLINE bool mem_chunk(int fd, u64 base, u64 length)
 {
     u64 mmoffset = base % sysconf(_SC_PAGESIZE);
     void* mmp = mmap(NULL, mmoffset + length, PROT_READ, MAP_SHARED, fd, base - mmoffset);
-    return mmp != MAP_FAILED;
+    if (mmp == MAP_FAILED)
+        return false;
+    if (munmap(mmp, mmoffset + length) < 0) {
```

Comment:
Sorry, just one last nitpick, no need for the curly braces `{}`

Reply:
I'll delete these braces but personally I don't think there's a problem with these.

Reply:
It's not a functional change, just a stylistic one, but it is the dominant style of the project, so let's be consistent about it.